impeller/entity/entity_pass.cc

```diff
@@ -896,6 +896,13 @@ bool EntityPass::RenderElement(Entity& element_entity,
   return true;
 }
 
+namespace {
+bool IsAlphaClampedToOne(PixelFormat pixel_format) {
```

Comment:
I would consider making this one of the helper functions defined in formats.h around PixelFormat if we need to use it anywhere else.

Reply:
It would be nice for this to be the place where we have a doc comment explaining why plus is an advanced blend.

Reply:
done